DVM location for certs

milohuman opened the pull request:
If this doc is going to replace the deployment companion guide, it needs to specify where to provide certificates during deployment.

azurecla commented:
Hi __@clemensv__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
Thanks for the contribution. I'll send to the author for review.

dominicbetts commented:
Hi @clemensv, thanks for the contribution. We'll get this PR accepted.

tysonn commented:
Thanks for the update, it has been merged :)

covetocove committed:
DVM location for certs

If this doc is going to replace the deployment companion guide, it needs to specify where to provide certificates during deployment.
